ontologies/Journal_with_items.json

```diff
@@ -0,0 +1,147 @@
+{
+  "@context": "https://api.wellcomecollection.org/catalogue/v0/context.json",
+  "id": "298yw4g",
+  "type": "Work",
+  "identifiers": 
+    [
+      {
+        "label": "b13106818",
+        "authority": "Sierra"
+      }
+    ],
+  "label": "The lancet",
+	"alternativeTitle": 
+	[
+		{
+			"label": "Lancet British edition",
+			"type": "keyTitle"
+		},
+		{
+			"label": "Lancet (London, England)",
+			"type": "uniformTitle"
+		}
+	],	
+  "publishers": 
+    [
+      {
+        "label": "J. Onwhyn",
+        "type": "Agent"
+      }
+    ],
+  "placeOfPublication": 
+     [
+       {
+        "label": "London",
+        "type": "Place"
+      }
+    ],
+  "publicationDate": 
+    {
+      "label": "1823",
+      "type": "Period"
+    },
+  "designation": "Vol. 1, no. 1 (Oct 5, 1823)-",
+  "issn": 
+  	[
+  		{
+  			"label": "0140-6736"
+  		},
+  		{
+  			"label": "0023-7507"
+  		},
+		{
+  			"label": "1474-547X"
+  		}
+  ],
+  "workType": "Journals",
+  "format": "Periodicals",
+  "frequency": 
+  	[
+  		{
+  			"label": "Frequency: weekly",
+			"type": "current"
+  		}
+  	],
+  "subjects": 
+    [
+     {
+        "label": "Medicine",
+        "authority": "MeSH" 
+      },
+      {
+        "label": "Medicine",
+        "authority": "fast"
+      }
+    ],
+  "generalNotes":
+    [
+		{
+			"label": "Subtitle: <1917- , \"A journal of British and foreign medicine, surgery, obstetrics, physiology, chemistry, pharmacology, public health, and news\""
+		},
+		{
+			"label": "Issued 1842-1901 also in an American edition (called: London lancet, 1842-1892; Lancet, 1895-1897; New York lancet, 1898-1901) and in a North American edition, July 1966-"
+		},
+		{
+			"label": "Editors: <1834-1850>, Thomas Wakley; <1917- , Samuel Squire Sprigge; <-1990, Gordon Reeves; 1991- , Robin Fox"
+      	}
+    ],
+	"numbering": "Volume designation varies",
+	"supplement": "Has special issues: \"2000\", published in 1999; \"Perspectives\", published in 2000", 
+	"relatedMaterialNote": 
+		[
+			{
+				"label": "Available in print and electronic formats."
+			},
+			{
+				"label": "Related to: London lancet"
+			},
+			{
+				"label": "Related to: London lancet (New York, N.Y.)"
+			},
+			{
+				"label": "Related to: New York lancet (New York, N.Y.)"
+			},
+			{
+				"label": "Lancet (North American ed.)"
+			},
+			{
+				"label": "Lancet (Deutsche Ausg.)"
+			}
+		],
+	"creators":
```

Comment:
These should be more specific than creator. They are 'Authors, etc' according to the label and 'Editors' according to the 700 field